weld/pretty_print.rs

```diff
@@ -26,6 +26,12 @@ impl PrintableType for Type {
             Scalar(I64) => "i64".to_string(),
             Scalar(F32) => "f32".to_string(),
             Scalar(F64) => "f64".to_string(),
+            Vectorized(Bool) => "<? x bool>".to_string(),
+            Vectorized(I8) => "<? x i8>".to_string(),
+            Vectorized(I32) => "<? x i32>".to_string(),
+            Vectorized(I64) => "<? x i64>".to_string(),
+            Vectorized(F32) => "<? x f32>".to_string(),
+            Vectorized(F64) => "<? x f64>".to_string(),
```

Comment:
Added parsing. This may need to change in the future but it works for now (namely, since the length is not encoded in the AST, there is no way to specify literals like `<1, 2, 3, 4>`. I added a `broadcast` node so vectors composed of a single constant can be expressed (e.g,. `<1, 1, 1, 1>` can be expressed with `broadcast(1)`).